src/statuspage/cli.js

```diff
@@ -12,9 +12,13 @@
 
 const yargs = require('yargs');
 const fs = require('fs');
-const request = require('request-promise-native');
+const fetchAPI = require('@adobe/helix-fetch');
 const { getIncubatorName } = require('../utils');
 
+const { fetch } = process.env.HELIX_FETCH_FORCE_HTTP1
+  ? fetchAPI.context({ httpsProtocols: ['http1'] })
+  : fetchAPI;
+
```

Comment:
This is a dangerous workaround since the tests will run with `http1` and production code with `http2` i.e. we never run the code with `http2` fetch for the tests...
But I assume it is ok until `nock` is http2 friendly.